flask_restless/views.py

```diff
@@ -1058,6 +1061,31 @@ def _search(self):
         for preprocessor in self.preprocessors['GET_MANY']:
             preprocessor(search_params=search_params)
 
+        # resolve date-strings as required by the model
+        for param in search_params.get("filters", list()):
```

Comment:
Use single quotes for string literals to match the rest of the code. (This comment applies to a few other lines as well.)